src/box/__tests__/box.test.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { View } from 'react-native';
+import Box from './../box';
+
+describe('Box#render', () => {
+  let wrapper;
+
+  beforeEach(() => {
+    wrapper = shallow(<Box />);
+  });
+
+  it('should own a props style that matches the default style', () => {
+    const style = {
+      height: 40,
```

Comment:
What is your thoughts on having a square box instead of a rectangle one by default ? Like setting `{ width: 50, height: 50}` or `{ width: 40, height: 40 }` ?

What is the most common use case you have in mind ?